config

```diff
@@ -1,4 +1,16 @@
 ngx_addon_name=ngx_http_statsd
-HTTP_MODULES="$HTTP_MODULES ngx_http_statsd_module"
-NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/ngx_http_statsd.c"
-CORE_LIBS="$CORE_LIBS -lssl"
+
+if test -n "$ngx_module_link"; then
+    ngx_module_type=HTTP
+    ngx_module_name=ngx_http_statsd_module
+    ngx_module_srcs="$ngx_addon_dir/ngx_http_statsd.c"
+
+    . auto/module
+else
+    HTTP_MODULES="$HTTP_MODULES ngx_http_statsd_module"
+    NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/ngx_http_statsd.c"
+    CORE_LIBS="$CORE_LIBS -lssl"
```

Comment:
Why do you still need `-lssl`? I believe that `USE_OPENSSL=YES` already accomplishes that. Or is there an issue with older nginx versions?